packages/did/src/DidResolver/DefaultResolver.ts

```diff
@@ -59,15 +65,20 @@ async function queryFullDetailsFromIdentifier(
     ]
   }
 
+  const didUri = getKiltDidFromIdentifier(identifier, 'full', version)
```

Comment:
having the `didUri` outside of the return is cleaner to read. I would recommend doing this else where.

Reply:
https://github.com/KILTprotocol/sdk-js/pull/428/commits/548d2cd01b0d6c2530d4bcdb877575007774a8b9